staging/src/k8s.io/api/bulk/v1alpha1/register.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GroupName is the group name for this API.
+const GroupName = "bulk.k8s.io"
+
+// SchemeGroupVersion is group version used to register these objects
+var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+
+// Resource takes an unqualified resource and returns a Group qualified GroupResource
+func Resource(resource string) schema.GroupResource {
+	return SchemeGroupVersion.WithResource(resource).GroupResource()
+}
+
+var (
+	// TODO: move SchemeBuilder with zz_generated.deepcopy.go to k8s.io/api.
```

Comment:
This comment seems to be no longer valid - this seems already to be in k8s.io/api
[How this works is that this staging/src/k8s.io/<repo>/ is periodically snapshotted and published as k8s.io/<repo> repository.]

Reply:
Such comment exists in all 'register.go' files (I just copied it).
It refers to the cleanup after removing of '/staging' (which is a temporal stage of moving parts of kubernetes into separate repos).

~/g/s/k/kubernetes (bulk-watch) $ grep -r 'TODO: move SchemeBuilder with zz_generated.deepcopy.go to k8s.io/api' | wc -l
52